packages/fxa-shared/l10n/rtl-locales.json

```diff
@@ -0,0 +1,18 @@
+[
+  "ar",
+  "ckb",
+  "dv",
+  "he",
+  "he-IL",
+  "ks",
+  "ps",
+  "fa",
+  "syr",
+  "ur",
+  "ug",
+  "pa",
+  "pa-IN",
+  "pa-PK",
```

Comment:
@vpomerleau as we discussed on a call, Punjabi in India `pa-IN` is read LTR, while Punjabi in Pakistan `pa-PK` is read RTL. At the very least, `pa-IN` should be removed. @bcolsson what does `pa` default to? Does it choose between `pa-IN` and `pa-PK` or default to the former? I'm inclined to remove `pa` as well, just in case. And additionally, FWIW, Pontoon shows a small percentage of translation completion for `pa-PK` for Settings/Payments so I don't know if it will make much of a difference right now to keep or remove this one.

What do y'all think?

Reply:
Oops! Missed @bcolsson 's message above: https://github.com/mozilla/fxa/pull/15941#discussion_r1366220992 . Keeping for historical reasons but thank you @bcolsson for verifying! Cheers!